packages-new/components/range/src/components/osds-range/osds-range.scss

```diff
@@ -7,10 +7,18 @@
 @import '~@ovhcloud/ods-theming/ods-theme';
 
 :host {
-  display: inline-flex;
   flex-flow: column;
-  width: var(--width);
   position: relative;
+
+  @include osds-range-on-selected-host() {
+    width: 100%;
```

Comment:
should not be necessary, it's the default behaviour for flex

Reply:
There is a "width: inherit" at line 53 that change the width of the actual range (not the container)